ReactWindows/ChakraBridge/ChakraHost.cpp

```diff
@@ -4,399 +4,428 @@
 
 void ThrowException(const wchar_t* szException)
 {
-	// We ignore error since we're already in an error state.
-	JsValueRef errorValue;
-	JsValueRef errorObject;
-	JsPointerToString(szException, wcslen(szException), &errorValue);
-	JsCreateError(errorValue, &errorObject);
-	JsSetException(errorObject);
+    // We ignore error since we're already in an error state.
+    JsValueRef errorValue;
+    JsValueRef errorObject;
+    JsPointerToString(szException, wcslen(szException), &errorValue);
+    JsCreateError(errorValue, &errorObject);
+    JsSetException(errorObject);
 };
 
 JsErrorCode DefineHostCallback(JsValueRef globalObject, const wchar_t *callbackName, JsNativeFunction callback, void *callbackState)
 {
-	JsPropertyIdRef propertyId;
-	IfFailRet(JsGetPropertyIdFromName(callbackName, &propertyId));
+    JsPropertyIdRef propertyId;
+    IfFailRet(JsGetPropertyIdFromName(callbackName, &propertyId));
 
-	JsValueRef function;
-	IfFailRet(JsCreateFunction(callback, callbackState, &function));
-	IfFailRet(JsSetProperty(globalObject, propertyId, function, true));
+    JsValueRef function;
+    IfFailRet(JsCreateFunction(callback, callbackState, &function));
+    IfFailRet(JsSetProperty(globalObject, propertyId, function, true));
 
-	return JsNoError;
+    return JsNoError;
 };
 
+JsErrorCode Stringify(JsValueRef value, ChakraHost* self, const wchar_t** szResult, size_t* sResult)
```

Comment:
Maybe we should just use JsConvertToString for all types during logging?  That's probably how it works in node, correct?